sgf-viewer/src/App.svelte

```diff
@@ -108,6 +123,12 @@
         height: 1em;
         width: 1em;
     }
+    @media (min-width: 992px) {  /* 992 === bootstrapLargeBreakpoint  */
```

Comment:
Can we use `@include media-breakpoint-up(lg) { ... }` here instead to avoid a magic number or does that include not work here for some reason?

Reply:
We can but that's SCSS rather than plain CSS, so we need to add an additional preprocess step to `svelte.config.js`. This works fine but actually slows down compile time a decent amount if done naively (by having the preprocess step import all of bootstrap so that this bootstrap mixin `media-breakpoint-up` is available). I'll leave that refactor to a different PR